theopensystemslab/planx-new
# feat: Wire up design settings (theme colour & logo)

## What does this PR do?
 - Split up `DesignSettings` component into reusable components
 - Wire up theme and logo form. This PR is primarily concerned with updating values in DB and `primaryColour` in theme, but is implemented throughout the application consistently in https://github.com/theopensystemslab/planx-new/pull/2658
 - `SHOW_TEAM_SETTINGS` feature flag still required

## To test
 - I can update theme colour
 - I can update logo
 - Preview is "live" and works as expected
 - "Reset" button works as expected

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/42eeccc3-ac38-46cf-aa08-1cc36b5760f4)
